template.distillery/demo_calendar.eliom

```diff
@@ -21,6 +21,16 @@ let%server service =
 let%client service = ~%service
 
 (* A reactive value containing the currently selected date *)
+(* NOTE: in this example, we want the signal to be stored on the server side,
+ * so that any change will be sent to the server. Only declaring this signal as
```

Comment:
That's not very accurate. The shared signal is defined on the server so that we can use it to produce the first page (and along with it, send a real signal that the client will use to update the page). This has nothing to do with sending changes to the server.